.eslintrc.json

```diff
@@ -2,14 +2,18 @@
   "extends": [
     "next",
     "next/core-web-vitals",
```

Comment:
I've removed the redundant  dependency from the "extends" array since `next/core-web-vitals` already includes most of them.  Moved `prettier` to the end  to  ensures that Prettier's formatting rules take precedence over any conflicting ESLint rules, resulting in a consistent code style. If any  of eslint issues  occurs in the future, we may consider to move  "next" to the end of the file as well  @WitchDevelops